pkg/bootstrap/bootstrap_plain_git.go

```diff
@@ -276,6 +288,19 @@ func (b *PlainGitBootstrapper) ReconcileSyncConfig(ctx context.Context, options
 		return err
 	}
 
+	sshUsername, err := b.promptSSHUsername(ctx)
```

Comment:
Could do with some documentation as to why this has been done and what providers require this.